### ISSUE TITLE
Issue with Recyclarr

### ISSUE THREAD

**Issue by dvdgorila** [RandomNinjaAtk/docker-radarr-extended#17]
Hello, 

Thanks for making this extended version of Radarr. I'm currently setting it up as a replacement for my radarr instance. 

One of the reasons i was compelled to test this out is because of the auto configuration/sync from trash guides, ie auto configuring custom formats

Its one of the first things i noticed custom format setting upon the initial run, they have not been configured. Based on the following Logs, there appears to be an issue. 

`022-10-03 04:51:29 :: Recycalarr :: Importing default recylarr config file to: /config/extended/configs/recyclarr.yaml
2022-10-03 04:51:29 :: Recycalarr :: Updating Radarr via Recyclarr
Found Unrecognized YAML Property: Failed to create an instance of type 'System.String'.
Please remove the property quoted in the above message from your YAML file
Exiting due to invalid configuration

2022-10-03 04:51:50 :: Recycalarr :: Complete
2022-10-03 06:42:00 :: Recycalarr :: Updating Radarr via Recyclarr
Found Unrecognized YAML Property: Failed to create an instance of type 'System.String'.
Please remove the property quoted in the above message from your YAML file
Exiting due to invalid configuration

2022-10-03 06:42:09 :: Recycalarr :: Complete
2022-10-03 08:18:59 :: Recycalarr :: Updating Radarr via Recyclarr
Found Unrecognized YAML Property: Failed to create an instance of type 'System.String'.
Please remove the property quoted in the above message from your YAML file
Exiting due to invalid configuration

2022-10-03 08:19:10 :: Recycalarr :: Complete
2022-10-03 08:19:32 :: Recycalarr :: Updating Radarr via Recyclarr
Found Unrecognized YAML Property: Failed to create an instance of type 'System.String'.
Please remove the property quoted in the above message from your YAML file
Exiting due to invalid configuration`

The only two things i veered from where the original storage directory. 

I'm using /data instead of /storage (based on trash guide recommendations and my experience with unraid i stay away from dropping everything under config folder for any data/media. 

The second was changing the standard webui redirect port from 7878 to 7979 because of the other instance im currently running. 

Let me know if i can provide any further details.

**Comment by RandomNinjaAtk** [comment]
Did you modify the default yaml?  If not, if you changed the port within Radarr, that may be the issue, because you shouldn't have to change it there.... Either way, its complaining about the yaml file, so feel free to open and edit it to get it working...

**Comment by dvdgorila** [comment]
Oh, i modified the external port, not the port within the container. Thanks for pointing me in the right direction. I had to update the Recycalarr yaml with the port and api key. Not sure why but originally listed the url as 127.0.0.1:null and the api key value as null. 

After changing that I was able to sync the custom formats. 

[INF] Total of 117 custom formats were synced
[INF] Updated 1 profiles and a total of 94 scores
2022-10-03 13:26:30 :: Recycalarr :: Complete
